src/pages/Login/Login.js

```diff
@@ -1,7 +1,102 @@
+import { useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
 import './Login.scss';
 
 function Login() {
-  return <div>Login</div>;
+  const [id, setId] = useState(' ');
+  const [pw, setPw] = useState(' ');
+  const navigate = useNavigate();
+
+  const getUserId = e => {
+    setId(e.target.value);
+  };
+
+  const getUserPw = e => {
+    setPw(e.target.value);
+  };
```

Comment:
1. 객체에 원하는 값에 접근하는 방법
2. spread 연산자
3. computed property
4. event 객체에서 이벤트가 일어나는 태그의 속성 가져오는 방법.
5. 같은함수를 2가지 input 이벤트 핸들러에 적용

이 순서로 잘 살펴보시기 바랍니다.
```suggestion
  const [userInfo, setUserInfo] = useState({id:"", pw:""});
  const navigate = useNavigate();

  const getUserInfo = e => {
     const {name, value} = e.target
  setUserInfo({...userInfo, [name]:value})
  };
```